configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,90 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
+  - dtc
+  - edk2
+  - fuse-sshfs
+  - gperftools
+  - gtk-vnc
+  - hexedit
+  - hivex
+  - icoutils
+  - ipxe
+  - kernel
+  - libdnet
+  - libguestfs
+  - libguestfs-winsupport
+  - libiscsi
+  - libmspack
```

Comment:
@rwmjones should libmspack and libdnet be moved to https://github.com/minimization/content-resolver-input/blob/master/configs/sst_virtualization_cloud-3rdparty_hypervisors.yaml ?

If yes, I ll ask Vitaly to update the file.

Reply:
Probably best to ask Cathy any questions related to `open-vm-tools` and its dependencies.  Note that `xmlsec1` is another depdendency of this package.